yt_dlp/extractor/archiveorg.py

```diff
@@ -704,6 +704,23 @@ class YoutubeWebArchiveIE(InfoExtractor):
                 'thumbnail': 'https://web.archive.org/web/20160108040020if_/https://i.ytimg.com/vi/SQCom7wjGDs/maxresdefault.jpg',
                 'upload_date': '20160107',
             },
+        }, {
+            # dmuxed formats
+            'url': 'https://web.archive.org/web/20240922160632/https://www.youtube.com/watch?v=z7hzvTL3k1k',
+            'info_dict': {
+                'id': 'z7hzvTL3k1k',
+                'ext': 'webm',
+                'title': 'Praise the Lord and Pass the Ammunition (BARRXN REMIX)',
+                'description': 'md5:45dbf2c71c23b0734c8dfb82dd1e94b6',
+                'uploader': 'Barrxn',
+                'uploader_id': 'TheRockstar6086',
+                'uploader_url': 'https://www.youtube.com/user/TheRockstar6086',
+                'channel_id': 'UCjJPGUTtvR9uizmawn2ThqA',
+                'channel_url': 'https://www.youtube.com/channel/UCjJPGUTtvR9uizmawn2ThqA',
+                'duration': 125,
+                'thumbnail': r're:https?://.*\.(jpg|webp)',
+                'upload_date': '20201207',
+            },
```

Comment:
could do this, not necessary though
```suggestion
            },
            'params': {'format': 'bv'},
```